src/core/reducers/search.js

```diff
@@ -21,6 +21,7 @@ export type FiltersType = {|
   page?: string,
   query?: Object,
   sort?: string,
+  guid?: string,
```

Comment:
Do you really need to change this file?

Reply:
ah, that's because you use `SearchParams` here. I missed that, so I guess it is alright. (I did not expect this file to be changed)